### DISCUSSION TITLE
Need Ideas: Enemy Names

### DISCUSSION THREAD

**Discussion by yanniboi** [LPGameDevs/EditarrrPublic#44] [Ideas]
We need enemy names! :) Suggestions below :)

### Enemy 1
![Enemy 1](https://img.itch.zone/aW1nLzQyMDM3ODguZ2lm/original/NhGuj%2B.gif)

### Enemy 2
![Enemy 2](https://img.itch.zone/aW1nLzQyMDM3OTMuZ2lm/original/MW5olb.gif)

### Enemy 3
![Enemy 3](https://img.itch.zone/aW1nLzQyMDM3OTkuZ2lm/original/t3%2FZRs.gif)

### Enemy 4
![Enemy 4](https://img.itch.zone/aW1nLzQyMDM4NTYuZ2lm/original/YGJzOc.gif)

### Enemy 5
![Enemy 5](https://img.itch.zone/aW1nLzQyMDM4NjEuZ2lm/original/LlXrJi.gif)

**Comment by BastisBastis** [comment]
Enemy 1 could be Clawdia.

Enemy 2 could be Teethan.

Enemy 3 could be Starthur.

Enemy 4 could be Clambert.

Enemy 5 could be Pol' MacDonald

**Comment by Bolrik** [comment]
Enemy 1: Pincerella
Enemy 2: Finley
Enemy 3: Stellar
Enemy 4: Shelldon
Enemy 5: Toots
